Sources/Client/Views/Play/MetalView.swift

```diff
@@ -28,8 +28,7 @@ struct MetalView {
     mtkView.framebufferOnly = true
     mtkView.clearColor = MTLClearColorMake(0.65, 0.8, 1, 1) // Sky colour
     mtkView.drawableSize = mtkView.frame.size
-    mtkView.depthStencilPixelFormat = .depth32Float
-    mtkView.clearDepth = 1.0
+    mtkView.depthStencilPixelFormat = .depth32Float // No depth stencil for view
```

Comment:
If it doesn't need a stencil, shouldn't that be set to the default value (afaict the default is just no depth stencil at all but this sets it to 32 bit floats). Maybe if you don't create a texture it doesn't affect anything, I dunno

Reply:
Well, this one is more tricky. I did remove the depth stencil from mtkView by setting it to `.invalid` here, but it introduced Metal API Validation error. Not sure how it could affect different hardware, so I've kept the depth stencil here.

Reply:
Ok, seems sensible